src/System.Collections/tests/Performance/Perf.List.cs

```diff
@@ -186,6 +186,21 @@ public void Enumerator(int size)
         [InlineData(1000)]
         [InlineData(10000)]
         [InlineData(100000)]
+        public void SetCapacity(int size)
+        {
+            List<object> list = CreateList(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    for (int i = 0; i < 10000; i++)
+                    {
+                        list.Capacity = size + 1;
```

Comment:
The docs [here](https://msdn.microsoft.com/en-us/library/y52x03h2(v=vs.110).aspx) only mention that setting the capacity is O(n) where n is the new size. They did not mention the special case of if the new capacity == the old capacity, but thinking on it now there's no reason to reallocate a new internal array if the desired size is already met with the old array. I'll modify this test to set the capacity to a different value each iteration.